Jenkinsfile

```diff
@@ -45,25 +45,22 @@ pipeline {
                 """)
 
                 echo runShell("echo \"It took \$((\$(date +'%s') - \$(cat time.log))) seconds to run the tests\"")
+
+                sh "mkdir bin && mv _build/default/src/stanc/stanc.exe bin/stanc"
+                stash name:'ubuntu-exe', includes:'bin/stanc, notes/working-models.txt'
             }
             post { always { runShell("rm -rf ./*")} }
         }
         stage("Run end-to-end tests") {
-            agent {
-                dockerfile {
-                    filename 'docker/debian/Dockerfile'
-                    //Forces image to ignore entrypoint
-                    args "-u root --entrypoint=\'\'"
-                }
-            }
+            agent { label 'linux' }
             steps {
+                unstash 'ubuntu-exe'
                 sh """
-                   git clone -j${env.PARALLEL} --recursive --branch develop https://github.com/stan-dev/cmdstan
-                   cd cmdstan && make -j${env.PARALLEL} build && cd ..
-               """
+          git clone --recursive --depth 50 https://github.com/stan-dev/performance-tests-cmdstan
+                   """
                 sh """
-                   eval \$(opam env)
-                   CMDSTAN="`pwd`/cmdstan" dune runtest test/integration/good/code-gen
+          cd performance-tests-cmdstan
+          STANC=\$(readlink -f ../bin/stanc) ./compare-git-hashes.sh "stat_comp_benchmarks --tests-file ../notes/working-models.txt" develop stanc3-dev develop develop
```

Comment:
I can't say I understand any of what this is doing, but I'm just going to trust that it is right.

Reply:
That's okay with me, haha. I think if we were post-launch a code reviewer might just inspect the Jenkins output to be sure it was testing the right thing (in this case stan2 vs stanc3 for the list of models).

Reply:
I tried that but Jenkins is giving me a 404 when I try to look at the output.